renderdoc/android/android.cpp

```diff
@@ -323,6 +323,34 @@ bool CheckAndroidServerVersion(const rdcstr &deviceID, ABI abi)
   return false;
 }
 
+Process::ProcessResult ListPackages(const rdcstr &deviceID, const rdcstr &packageNames)
+{
+  Process::ProcessResult result =
+      adbExecCommand(deviceID, "shell getprop ro.build.version.release");
+
+  int androidVersion = -1;
+  if(result.retCode == 0)
+  {
+    char *strEnd = nullptr;
+    androidVersion = static_cast<int>(strtol(result.strStdout.c_str(), &strEnd, 10));
```

Comment:
1. This code is not executed on device, so you should not have any problems with `strtol`.
2. `strtol` is needed because `ro.build.version.release` is a string. [Docs](https://developer.android.com/reference/android/os/Build.VERSION#RELEASE) say it can be whatever it is. So we have to check whether the conversion is succeeded or not. If it's failed then `strEnd` will remain being null. With `atoi` there is no way to check it. Only `errno` maybe?

Reply:
Alright, we can check SDK level. It is always int. The minimal for android 13 is 33-ish.

Reply:
It was a build failure from what I remember, this code is still compiled on android (though indeed it doesn't need to be technically, but keeping it building is simpler than either conditionally compiling stubs or conditionally snipping out the dependencies elsewhere).

If it can be a string then we shouldn't use that property. You should maybe use `ro.build.version.sdk` like in the other places that check for android versions, as I believe there is a corresponding SDK value for every android version.